src/rubicon/objc/api.py

```diff
@@ -1016,40 +1092,40 @@ def __getattr__(self, name):
         """Allows accessing Objective-C properties and methods using Python attribute
         syntax.
 
-        If ``self`` has a Python attribute with the given name, its value is
+        If `self` has a Python attribute with the given name, its value is
         returned.
 
         If there is an Objective-C property with the given name, its value is
         returned using its getter method. An attribute is considered a property
         if any of the following are true:
 
         * A property with the name is present on the class (i.e. declared using
-          ``@property`` in the source code)
+          `@property` in the source code)
         * There is both a getter and setter method for the name
         * The name has been declared as a property using
-          :meth:`ObjCClass.declare_property`
+          [`ObjCClass.declare_property`][rubicon.objc.api.ObjCClass.declare_property]
```

Comment:
Since this is a `:meth:` reference, it should probably have the `()`.
```suggestion
          [`ObjCClass.declare_property()`][rubicon.objc.api.ObjCClass.declare_property]
```